Gemfile.lock

```diff
@@ -1,13 +1,3 @@
-GIT
```

Comment:
hah, here is the gem with sinatra. :) Still wondering if maybe these changes should be in a separate PR?